example/docker-compose/distributed/docker-compose.yaml

```diff
@@ -157,17 +157,3 @@ services:
       - GF_FEATURE_TOGGLES_ENABLE=traceqlEditor
     ports:
       - "3000:3000"
-
-  agent:
-    image: grafana/agent:v0.38.1
-    volumes:
-      - /tmp/agent:/tmp/agent
-      - ./agent.river:/etc/agent.river
-    environment:
-      - AGENT_MODE=flow
-    entrypoint:
-      - sh
-      - -euc
-      - /bin/grafana-agent run --server.http.listen-addr=0.0.0.0:12345 /etc/agent.river
-    ports:
-      - "12345:12345"
```

Comment:
The agent is used when using self-monitoring for batching traces and preventing a feedback loop (see https://github.com/grafana/tempo/pull/4031/files#diff-d780bf1545ede1bdf49ea1419cc0139b44f70b0cbfafc9493df7ffa49763a1a5L109-L113). We should replace it with alloy.

Reply:
ahh I see, I missed those comments. Good catch!